database/seedDatabase.js

```diff
@@ -0,0 +1,114 @@
+const mongoose = require('mongoose');

```

Comment:
I think you should separate your concerns better for the database and clarify the file naming. Your seeding script defines your review schema, when it should just populate it with data.